### PR TITLE
new package: readline 8.0p004

### PR DESCRIPTION
**Pull request by a-schaefers**
Read: https://k1ss.org/style
Read: https://k1ss.org/guidestones

---

## Description of package
Turns out I need this package to make the ruby irb repl function. So I figure others will end up needing this package for something or other sooner or later. Add it to my list!

Note: 8.0p004 or 8.0_p4 or 8.0p4 all seem to be acceptable repology version naming conventions.

Note: I also chose to download the "patches" from the official gnu upstream source, (instead of keeping a local copy), and because they are official patches, I believe this makes sense.

## New package

- [x] Latest upstream version.
- [x] I agree to maintain this package.
- [x] I agree to notify @dylanaraps if I can no longer maintain this package.

## Existing package

- [x] I am the maintainer of this package.

### CODE REVIEW

File: community/readline/build [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+for patch in readline80-00* ; do patch -p0 < "$patch"; done
```
Comment (dylanaraps) [inline community/readline/build]: https://k1ss.org/style.txt 1800

File: community/readline/build [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+./configure \
+    --prefix=/usr
+
+make DESTDIR="$1" install
```
Comment (dylanaraps) [inline community/readline/build]: https://k1ss.org/style 0400

File: community/readline/build (hunk 1 above)
Comment (a-schaefers) [reply community/readline/build]: In this case the files are staying on the upstream official source, I would like to request a style guide exception.

Comment (dylanaraps) [reply community/readline/build]: I meant breaking it onto multiple lines. See 1800.

Comment (a-schaefers) [reply community/readline/build]: oh, okay

Comment (a-schaefers) [reply community/readline/build]: I thought it was the glob, the problem

**Pull request merged**